RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -5,6 +5,15 @@
 # run only if there are high pT jets
 jetsForCoreTracking = cms.EDFilter("CandPtrSelector", src = cms.InputTag("ak4CaloJetsForTrk"), cut = cms.string("pt > 100 && abs(eta) < 2.5"))
 
+from RecoEgamma.EgammaPhotonProducers.egammaForCoreTracking_cff import particleFlowSuperClusterECALForTrk, egammasForTrk, egammaHoverEForTrk, egammasForCoreTracking, egammaForCoreTrackingTask
+egammaAndJetsForCoreTracking = cms.EDProducer( "CandidateCombiner",
+                                               coll1 = cms.InputTag("egammasForCoreTracking"),
+                                               coll2 = cms.InputTag("jetsForCoreTracking"),
+                                               maxDRToClean = cms.double(0.05)
+)
+
+#egammaForCoreTrackingSeq = cms.Sequence(egammaCandidates * egammaHoverE * egammasForCoreTracking)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
removed